### DISCUSSION TITLE
Allot of Requests in Top Paths for a specific URL

### DISCUSSION THREAD

**Discussion by MikevPeeren** [vercel/community#846] [Help] [answered]
I am wondering if unknowingly to me we have captured a form of DDOS or that it is a bug regarding allot of requests for the Top Paths section: 

Allot of requests to entry:
<img width="1334" alt="Screenshot 2022-09-13 at 09 29 11" src="https://user-images.githubusercontent.com/15649037/189839636-1681fca3-9f25-4c70-b400-da876292a38d.png">

Normal day:
<img width="1334" alt="Screenshot 2022-09-13 at 09 29 33" src="https://user-images.githubusercontent.com/15649037/189839998-5435c577-2bbf-4092-9c77-f5c9194afccd.png">

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @MikevPeeren. Sorry you're dealing with this. Based on the pattern, it does seem like a possible DDoS attack. If you want more detailed information about your account usage or any other help related to this issue, you should visit [vercel.com/help](https://vercel.com/help) and submit a new case for the Customer Success Team. That will allow you and the team to share more specific details with each other while keeping your information private.

**Comment by leerob** [comment]
Hey, wanted to follow up here.

We just launched **[Attack Challenge Mode](https://vercel.com/changelog/prevent-malicious-traffic-with-attack-challenge-mode-for-vercel-firewall)** –an extra layer of defense for DDoS and malicious traffic, free for all plans. When enabled, the Vercel Firewall will serve a challenge page to help mitigate attacks.

Hope this helps!
